### DISCUSSION TITLE
Incomplete articles saved

### DISCUSSION THREAD

**Discussion by arhadrian** [omnivore-app/omnivore#2769] [Q&A]
I have many attempts to save articles that only saved the beginning of the text. Is this a bug?
I noticed one thing: that page, if in the reading mode in Firefox, also displays only the beginning of the text.
I wouldn't like to go back to the classic copy-and-paste solution to save content... I repeatedly reported this in the app, but no answer. Thank you.

**Comment by jacksonh** [comment]
Hi, when you submit in-app we anonymize the reports in case someone accidentally submitted something private, so we can't reply directly to those reports. 

Do you have an example of one that only shows the beginning on Firefox?
